src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -41,4 +43,20 @@
         networkModificationService.changeSwitchState(networkUuid, switchId, open);
         return ResponseEntity.ok().build();
     }
+
+    @PostMapping(value = "/networks/{networkUuid}/{equipmentType}/{equipmentId}")
```

Comment:
should be a PUT instead of a POST, like for switches ....

Reply:
yes it has a body